src/DevChatter.Bot.Core/Data/Model/RouletteSettingsEntity.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DevChatter.Bot.Core.Data.Model
+{
+    public class RouletteSettingsEntity : DataEntity
+    {
+        public int WinChance { get; set; }
```

Comment:
Let's rename this to make it clear that it's a percentage number (assuming it is). Something like `WinPercentChance`.